network/src/commonMain/kotlin/com/wire/kalium/network/api/v3/authenticated/ConversationApiV3.kt

```diff
@@ -18,8 +27,39 @@ internal open class ConversationApiV3 internal constructor(
             )
         }
 
-    protected companion object {
+    override suspend fun fetchGlobalTeamConversationDetails(selfUserId: UserId, teamId: TeamId): NetworkResponse<ConversationResponse> {
+        return wrapKaliumResponse<GlobalTeamConversationResponse> {
+            httpClient.get("$PATH_TEAM/$teamId/$PATH_CONVERSATIONS/$PATH_GLOBAL")
+        }.mapSuccess { response ->
```

Comment:
Why mapping manually?

Reply:
§GlobalTeamConversationResponse` isn't compatible with the `ConversationResponse` so filling out some fake values to make it compatible.

This is all a temporary solution though because how the global team conversation is retrieved will be refactoring by the BE in the next sprint.